addons/editor-dark-mode/addon.json

```
@@ -1139,6 +1152,7 @@
         "primary": "#5498c7",
         "highlightText": "#21211f",
         "menuBar": "#c0c3c6",
+        "popup": "#5498c7e6",
```

```suggestion
        "popup": "#00000099",
```
Transparent black would look better in this preset too.